quiz.js

```diff
@@ -1,168 +1,137 @@
     let questionIndex = 0;
     let correctAnswer;
 
     //audio variables
-
-    //crowd cheer sound
     const cheer = new Audio('./Assets/audio/bbc_crowds--ch_07043047.mp3');
-    //crow jeer sound
-    const jeer = new Audio("./Assets/audio/bbc_crowd-reac_07018034.mp3")
-    // MF DOOM backtrack
-    const DOOM = new Audio("./Assets/audio/Metal Fingers Special Herbs (Volume 1&2) [FULL ALBUM].mp3")
+    const jeer  = new Audio("./Assets/audio/bbc_crowd-reac_07018034.mp3")
+    const DOOM  = new Audio("./Assets/audio/Metal Fingers Special Herbs (Volume 1&2) [FULL ALBUM].mp3")
+
+    //call ApiRequest() function to start 
+    sendApiRequest();
 
     //Api request
     async function sendApiRequest() {
         // set response equal to response from url using fetch method
         let response = await fetch(`https://opentdb.com/api.php?amount=10&type=multiple`);
         // Takes a Response stream and reads it to completion. It returns a promise that resolves with the result of parsing the body text as JSON, which is a JavaScript value of datatype object, string, etc. *** from MDN
         let data = await response.json();
-        //set questions equal to all returned data using the spread operator
         questions = [...data.results];
-        //call generateQuestion function
         generateQuestion();
-        console.log(questions)
     }
-    //this function generates new questions
-    function generateQuestion() {
 
-        //if statement to call endGame function once questions.length is reached
+    //this function generates the next question
+    function generateQuestion() {
+        //If we've gone through all the questions, end the game
         if (questionIndex === questions.length) {
             endGame();
         }
 
-        //emptys answer buttons
-        answerZoneEl.empty();
-        //emptys right/wrong area
-        rightWrongEl.empty();
-
-        cheer.pause();
-        jeer.pause();
-        function fix (errorString){
-            if(errorString.includes("&#039;")){
-                console.log("its in there")
-            }
-            else {(errorString.includes("&#039;"))
-                    console.log("its not there")
-            }
-            while(errorString.includes("&#039;")){
-                errorString = errorString.replace("&#039;", "\'")
-                }
-            while(errorString.includes("&quot;")){
-                errorString = errorString.replace("&quot;","\"")
-                }
-            while(errorString.includes("&amp;")){
-                errorString = errorString.replace("&amp;","&")
-            }
-            while(errorString.includes("&Uuml;")){
-                errorString = errorString.replace("&Uuml;","Ü")
-            }    
-            while(errorString.includes("&eacute;")){
-                errorString = errorString.replace("&eacute;","é")
-            }        
-            return errorString
-        }
-        //sets currentQuestion equal to questions index
```

Comment:
Other encoded characters exist outside of these. One solution I found on stackoverflow takes the string, inserts it into a DOM element which decodes all these, and then gets it back off of that DOM element in its decoded format. See `fix(...)`